bin/pycbc_live

```diff
@@ -120,6 +120,12 @@ class LiveEventManager(object):
                                 available_cores)
                 self.fu_cores = 1
 
+        if args.enable_embright_has_massgap:
+            # check if HasMassGap is enabled and set mass gap max limit
```

Comment:
I think the line of comment is not needed, the code is basically self explanatory